src/seedsigner/views/seed_views.py

```diff
@@ -1226,6 +1244,249 @@ def run(self):
+
+        elif button_data[selected_menu_num] == self.EXPORT_SCAN_PRIVKEY:
+            return Destination(SeedBIP352ExportScanPrivkeyDetailsView, view_args={"seed_num": self.seed_num})
+
+
+
+class SeedBIP352ExportSpendPubkeyDetailsView(View):
+    def __init__(self, seed_num: int):
+        super().__init__()
+        self.seed_num = seed_num
+        self.seed = self.controller.get_seed(seed_num)
+        self.network = self.settings.get_value(SettingsConstants.SETTING__NETWORK)
+        self.master_fingerprint = self.seed.get_fingerprint(network=self.network)
+        self.spend_pubkey = self.seed.derive_bip352_spend_privkey(network=self.network).to_public().to_string()
+        self.derivation_path = self.seed._build_bip352_path(is_scan_key=False, network=self.network)
+        
+
+    def run(self):
+        selected_option = self.run_screen(
+            BaseKeyDetailsScreen,
+            title=_("Spend Pubkey Details"),
+            key_label=_("Spend Public Key"),
+            key_value=self.spend_pubkey,
+            fingerprint=self.master_fingerprint,
+            has_passphrase=self.seed.has_passphrase,
+            derivation_path=self.derivation_path,
+            button_data=[ButtonOption("Export Pubkey via QR")],
+            icon_name=FontAwesomeIconConstants.X
+        )
+
+        if selected_option == RET_CODE__BACK_BUTTON:
+            return Destination(BackStackView)
+        
+        return Destination(SeedBIP352ExportSpendPubkeyQRView, view_args={"seed_num": self.seed_num})
+
+
+
+class SeedBIP352ExportScanPrivkeyDetailsView(View):
+    def __init__(self, seed_num: int):
+        super().__init__()
+        self.seed_num = seed_num
+        self.seed = self.controller.get_seed(seed_num)
+        self.network = self.settings.get_value(SettingsConstants.SETTING__NETWORK)
+        self.master_fingerprint = self.seed.get_fingerprint(network=self.network)
+        self.scan_privkey = self.seed.derive_bip352_scan_privkey(network=self.network).to_string()
+        self.derivation_path = self.seed._build_bip352_path(is_scan_key=True, network=self.network)
+
+
+    def run(self):
+        if self.settings.get_value(SettingsConstants.SETTING__DIRE_WARNINGS) != SettingsConstants.OPTION__DISABLED:
+            selected_warning_option = self.run_screen(
+                WarningScreen,
+                title=_("Caution!"),
+                status_headline=_("Privacy Leak!"),
+                text=_("Your scan private key can be used to detect all your silent payments but cannot spend funds."),
+                button_data=[ButtonOption("Continue")]
+            )
+            if selected_warning_option == RET_CODE__BACK_BUTTON:
+                return Destination(BackStackView)
+
+        selected_details_option = self.run_screen(
+            BaseKeyDetailsScreen,
+            title=_("Scan Private Key Details"),
+            key_label=_("Scan Private Key"),
```

Comment:
`TRANSLATOR_NOTE` - this one could be misinterpreted as a command ("Scan your private key now" instead of talking about the "scan private thing" as a noun). Have to make sure the translators aren't confused about the intention here.